CMakeLists.txt

```diff
@@ -164,7 +165,7 @@ if (TI_WITH_LLVM)
       set (CLANG_EXECUTABLE ${CMAKE_CXX_COMPILER})
       message("Clang executable using host compiler ${CLANG_EXECUTABLE}")
     else()
-      find_program(CLANG_EXECUTABLE NAMES clang-15 clang-14 clang-13 clang-12 clang-11 clang-10 clang-9 clang-8 clang-7 clang)
+      find_program(CLANG_EXECUTABLE NAMES clang-21 clang-20 clang-19 clang-18 clang-17 clang-16 clang-15 clang-14 clang-13 clang-12 clang-11 clang-10 clang-9 clang-8 clang-7 clang)
```

Comment:
again: looks scary to me. What are your thoughts on how we can test this?

Reply:
(Like: in the CI somehow)

Reply:
(should we drop some of the older versions perhaps? 🤔 )

Reply:
i thought that also hahaha